src/util/misc/download.ts

```diff
@@ -22,3 +26,20 @@ export function downloadFileAndSave(downloadUrl: string, filePath: string): Prom
       .on("finish", () => resolve()));
   });
 }
+
+export async function downloadArtifacts(command: AppCommand, streamingOutput: StreamingArrayOutput, outputDir: string, testRunId: string, artifacts: { [propertyName: string]: string }): Promise<void> {
+  for (const key in artifacts) {
+
+    const reportPath: string = fsHelper.generateAbsolutePath(outputDir);
+    const pathToArchive: string = path.join(reportPath, `${key.toString()}.zip`);
+    fsHelper.createLongPath(reportPath);
+    await downloadFileAndSave(artifacts[key], pathToArchive);
+
+    // Print only in VSTS environment
+    if (process.env["TF_BUILD"]) {
```

Comment:
is `TF_BUILD` always present in VSTS and TFS?  IS there a document that describes this somewhere?

Reply:
@krukow 
https://docs.microsoft.com/en-us/vsts/build-release/concepts/definitions/build/variables?view=vsts&tabs=batch#tfbuild

Also I've asked about it in a channel related to VSTS

Reply:
@erychagov great - please add the link to the code comment